src/lib/Components/Bidding/Screens/Registration.tsx

```diff
@@ -123,15 +123,15 @@ export class Registration extends React.Component<RegistrationProps, Registratio
   }
 
   render() {
-    const { live_start_at, end_at } = this.props.sale
+    const { live_start_at, end_at, is_preview, start_at } = this.props.sale
     const { billingAddress, creditCardToken: token } = this.state
 
     return (
       <BiddingThemeProvider>
         <Container m={0}>
           <Flex alignItems="center">
             <Title mb={3}>Register to bid</Title>
-            <Timer liveStartsAt={live_start_at} endsAt={end_at} />
+            <Timer liveStartsAt={live_start_at} endsAt={end_at} isPreview={is_preview} startsAt={start_at} />
```

Comment:
#minor and not related to the story but I wonder if there's any simple way to convert `snake_case` keys to `CamelCase`.